index.html

```diff
@@ -6,25 +6,52 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <title>Awesome Books</title>
+    <script src="https://cdn.jsdelivr.net/npm/luxon@2.0.2/build/global/luxon.min.js"></script>
+    <script src="/js/time.js" defer></script>
     <script src="/index.js" defer></script>
+    <script src="/js/navigation.js" defer></script>
 </head>
 <body>
-    <header></header>
+    <div class="wrapper"></div>
+    <header>
+        <nav class="nav">
+            <span class="nav-title">Awesome Books</span>
+            <div class="nav-items">
+                <a class="list" href="#">List</a>|
+                <a class="add-new" href="#">Add new</a>|
+                <a class="contact" href="#">Contact</a>
```

Comment:
- [ ] Optional
When the user navigating to a specific page, we can change the colour of that nav link connected to that specific page. Let's try to improve this.